contrib/backport.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -e
+shopt -s extglob # enable extended pattern matching features
+
+
+#############
+# VARIABLES #
+#############
+
+stable_branch=$1
+commit=$2
+bkp_branch_name=$3
+bkp_branch_name_prefix=bkp
+bkp_branch=$bkp_branch_name-$bkp_branch_name_prefix
+
+
+#############
+# FUNCTIONS #
+#############
+
+git_status () {
+  if [[ $(git status --porcelain | wc -l) -gt 0 ]]; then
+    echo "It looks like you have not committed changes:"
+    echo ""
+    git status --short
+    echo ""
+    echo ""
+    echo "Press ENTER to continue or Ctrl+C to break."
+    read -r
+  fi
+}
+
+checkout () {
+  git checkout -b "$bkp_branch" origin/"$stable_branch"
+}
+
+cherry_pick () {
+  local x
+  for com in ${commit//,/ }; do
+    x="$x $com"
+  done
+  # trim the first white space and use an array
+  # for more info see: https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions
+  x=(${x##*( )})
+  git cherry-pick --allow-empty-message -s "${x[@]}"
```

Comment:
Would you please add `-x` to the `cherry-pick` command, so the original sha1 is preserved in the log?

Why would we want to allow empty commit messages?